Code/GraphMol/RGroupDecomposition/RGroupFingerprintScore.cpp

```diff
@@ -62,7 +62,7 @@ void addFingerprintToRGroupData(RGroupData *rgroupData) {
     try {
       MolOps::sanitizeMol(mol);
     } catch (MolSanitizeException &) {
-      BOOST_LOG(rdWarningLog)
+      BOOST_LOG(rdDebugLog)
```

Comment:
I'm happy to merge as-is, but it seems to me like this is a warning and not a debug message

Reply:
The experience I'm having is that the warning is common and not unexpected given that the group is a fragment of a structure. We still fingerprint when the sanitization fails, so I thought that a debug message would be more appropriate.  Happy for it to be left a warning if you disagree.

Reply:
> The experience I'm having is that the warning is common and not unexpected given that the group is a fragment of a structure. We still fingerprint when the sanitization fails, so I thought that a debug message would be more appropriate. Happy for it to be left a warning if you disagree.

Na, that makes sense to me. I just wanted to check in on it.